modules/lb/main.tf

```diff
@@ -52,7 +52,7 @@ resource "azurerm_lb_backend_address_pool" "bepool" {
 resource "azurerm_lb_probe" "this" {
   name                = "${var.name}-lb-cp-probe"
   loadbalancer_id     = azurerm_lb.this.id
-  resource_group_name = data.azurerm_resource_group.rg.name
+# resource_group_name = data.azurerm_resource_group.rg.name
```

Comment:
Do not leave in commented out code. If it should be removed just delete it.

Reply:
Thanks @Nepomuceno cleaned up the code